optuna/_runtime_info.py

```diff
@@ -0,0 +1,34 @@
+import platform
+import sys
+from typing import Optional
+from typing import TextIO
+
+import optuna
+
+
+class _RuntimeInfo(object):
```

Comment:
I'm not sure if we should annotate this as `@experimental`.
It would be noisy when users use it when opening an issue.

```
> python -c 'import optuna; optuna.print_runtime_info()'
/home/himkt/work/github.com/himkt/optuna/optuna/_runtime_info.py:25: ExperimentalWarning: _RuntimeInfo is experimental (supported from v2.9.0). The interface can change in the future.
  return _RuntimeInfo()
- Optuna version: 2.9.0.dev
- Python version: 3.9.5
- OS: Linux-5.4.72-microsoft-standard-WSL2-x86_64-with-glibc2.31
```